ml_genn/layers/avepool2d_conv2d_synapses.py

```diff
@@ -95,14 +95,24 @@ def __init__(self, filters, pool_size, conv_size, pool_strides=None,
                  conv_padding='valid', connectivity_type='procedural'):
         super(AvePool2DConv2DSynapses, self).__init__()
         self.filters = filters
-        self.pool_size = pool_size
-        self.conv_size = conv_size
-        if pool_strides == None:
-            self.pool_strides = (pool_size[0], pool_size[1])
+        if isinstance(pool_size, int):
+            self.pool_size = (pool_size, pool_size)
+        else:
+            self.pool_size = pool_size
```

Comment:
this could perhaps go in a helper function as I suspect it's fairly ubiquitous 
```python
def getParam2D(param, default=None):
    if param is None:
        if default is not None:
            return default
        else:
            raise Exception("Parameter required")
    elif isinstance(param, int):
        return (param, param)
    else:
        return param
```

Reply:
Done.